packages/terra-core-docs/src/terra-dev-site/doc/show-hide/About.1.doc.mdx

```diff
@@ -1,19 +1,28 @@
 import { Badge } from 'terra-show-hide/package.json?dev-site-package';
+import { Notice } from "@cerner/terra-docs";
 
 import DefaultShowHide from './example/DefaultShowHide?dev-site-example';
 import CustomButtonTextShowHide from './example/CustomButtonTextShowHide?dev-site-example';
+import LinkShowHide from './example/LinkShowHide?dev-site-example';
 import InitiallyOpenShowHide from './example/InitiallyOpenShowHide?dev-site-example';
 import ButtonAlignCenterShowHide from './example/ButtonAlignCenterShowHide?dev-site-example';
 import ButtonAlignRightShowHide from './example/ButtonAlignRightShowHide?dev-site-example';
 import NoPreviewShowHide from './example/NoPreviewShowHide?dev-site-example';
+import NoFocusRefShowHide from './example/NoFocusRefShowHide?dev-site-example';
 
 import ShowHidePropsTable from 'terra-show-hide/src/ShowHide?dev-site-props-table';
+import ShowHideFocuserPropsTable from 'terra-show-hide/src/ShowHideFocuser?dev-site-props-table';
 
 <Badge />
 
 # Terra Show Hide
 
-Show Hide Component that will show a preview of content and then expand it with a Show More button.
+Show Hide Component - is a disclosure widget that enables content to be either collapsed (hidden) or expanded (visible). It has two elements: a disclosure [button][1] and a section of content whose visibility is controlled by the button. When the controlled content is hidden, the button is often styled as a typical push button with a down-pointing caret to hint that activating the button will display additional content. When the content is visible, the content displays above the button and the caret points up.
+
+Another Terra component, [Toggle Button][2], is available and preferred to Show/Hide because it better adheres to accessibility best practices for disclosure components. It differs from Show Hide by displaying the content below the button when it is activated. 
```

Comment:
```suggestion
Another Terra component, [Toggle Button][2], is available and preferred to Show/Hide when possible because it better adheres to accessibility best practices for disclosure components. It differs from Show Hide by displaying the content below the button when it is activated. 
```